Update versioning info to point to the file format repo, closes #679 (#680)
sketch-hq/SketchAPI: docs/pages/file-format-versioning.md

## Version history

> For information about how `appVersion` maps to document `version`, and how they relate to the new file format schema, head over to the [sketch-file-format](https://github.com/sketch-hq/sketch-file-format/#sketch-document-version-mapping) repo.

Reviewer: Can be more direct I think, also the fact that the file format is new is irrelevant, instead should be seen as a matter of fact. Also, more generally, let's keep using "repository" instead of "repo".

```suggestion
See [sketch-file-format](https://github.com/sketch-hq/sketch-file-format/#sketch-document-version-mapping) documentation for information about how `appVersion` maps to document `version`.
```
